### PR TITLE
Rbc

### PR DESCRIPTION
**Pull request by psakievich**
Not intending to merge this right now.  Having an issue.  tracer dirichlet bc's don't seem to be enforced on the top and bottom of the cell.  Can someone take a look at this to help me see what I'm missing?

image

image

### CODE REVIEW

**Comment by psakievich** [comment]
@michaeljbrazell @jrood-nrel

File: src/setup/init_fluid.f90 [hunk 1]
```diff
@@ -1,468 +1,514 @@
...
+                        tracer(i,j,k,3) = 3.d0
+                end do
+            end do
+        end do
+
+    end subroutine channel_flow
+
+    ! Initialize potential temperature with a perturbation factor
+    ! RBC domain should have height of unity and be solved with non-dimensional variables
+    subroutine rayleigh_benard(lo, hi, vel, tracer, slo, shi, dx, dy, dz, xLo, yLo, zLo, domlo)
```
Comment (psakievich) [inline src/setup/init_fluid.f90]: Eclipse must have some fortran format modification that I haven't changed yet.  This routine is the only real change in this file.

**Comment by michaeljbrazell** [comment]
I don't think it is possible to specify a Dirichlet on a tracer yet. Look at subroutine` set_scal_diff_bc ()`

**Comment by sayerhs** [comment]
@psakievich I am closing this request as the code has changed quite a bit from when you submitted this.

**Pull request closed without merging**